azure/HandsOnTutorial/11-upload-file-to-blob.fsx

```diff
@@ -0,0 +1,79 @@
+﻿(*** hide ***)
+#load "credentials.fsx"
+
+open System
+open System.IO
+open MBrace.Core
+open MBrace.Store
+open MBrace.Azure
+open MBrace.Azure.Client
+open MBrace.Flow
+
+open Microsoft.WindowsAzure
+open Microsoft.WindowsAzure.Storage
+open Microsoft.WindowsAzure.Storage.Auth
+open Microsoft.WindowsAzure.Storage.Blob
+
+(**
+This tutorial illustrates how to upload local files to Azure blob storage and 
+process the files in the cloud using MBrace.
+
+Before running, edit credentials.fsx to enter your connection strings.
+*)
+
+(** First you connect to the cluster: *)
+let cluster = Runtime.GetHandle(config)
+
+cluster.ShowProcesses()
+cluster.ShowWorkers()
+
+
+(** Create a tmp file locally, wrote some random content, and return the file path. *)
```

Comment:
Please write dialog in the form "Now you ...", "Next you ..." etc. and use full English sentences like a book.

"The following function creates a temporary file locally, containing some random content, and returns the file path."

"Next you create a local temporary file."

"Next you use the Azure SDK to create a temporary container  to contain blobs."

etc.